ludwig/features/base_feature.py

```diff
@@ -301,7 +301,6 @@ def update_model_definition_with_metadata(
     @staticmethod
     @abstractmethod
     def calculate_overall_stats(
```

Comment:
here the name of the parameters should match the new convention: predictions, targets

Reply:
incorporated comment.